examples/medplum-task-example-app/app/src/pages/ResourcePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { Tabs, Title } from '@mantine/core';
+import { getDisplayString, getReferenceString } from '@medplum/core';
+import { Resource, ResourceType } from '@medplum/fhirtypes';
+import { DefaultResourceTimeline, Document, ResourceTable, useMedplum } from '@medplum/react';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { ResourceHistoryTab } from '../components/ResourceHistoryTab';
+
+/**
+ * This is an example of a generic "Resource Display" page.
+ * It uses the Medplum `<ResourceTable>` component to display a resource.
+ * @returns A React component that displays a resource.
+ */
+export function ResourcePage(): JSX.Element | null {
+  const medplum = useMedplum();
+  const { resourceType, id } = useParams();
+  const [resource, setResource] = useState<Resource | undefined>(undefined);
+  const tabs = ['Details', 'Timeline', 'History'];
+  const [currentTab, setCurrentTab] = useState<string>(() => {
+    const tab = window.location.pathname.split('/').pop();
+    return tab && tabs.map((t) => t.toLowerCase()).includes(tab) ? tab : tabs[0].toLowerCase();
+  });
+
+  useEffect(() => {
```

Comment:
This is probably based on old code. We can probably use the `useResource` hook here